openapi/docs/hosted_onboarding.md

```diff
@@ -1,29 +1,32 @@
+In order to process payments, each of your customers (whom we refer to as `businesses`) will have to be onboarded on our platform. Once they are added they go through an approval process. JustiFi's hosted onboarding provides you with an easy-to-implement, user-friendly way to collect the required business and financial information from each business within your platform. Once approved, your business can process payments through JustiFi.
 
-To onboard a new sub account via hosted onboarding
-1. Create a sub account
+To onboard a new business via hosted onboarding
+1. Create a business
 2. Include JustiFi hosted onboarding in your application
 3. (optional) Listen to success/fail message
-4. Check the sub account's status
+4. Check the underwriting status of the sub account connected to the business
 
 
-### Create a sub account
 
-#### Create a sub account
+### Create a business
+
+#### Create a business
 ```sh
 curl -X POST \
-  https://api.justifi.ai/v1/sub_accounts \
+  https://api.justifi.ai/v1/entities/business \
   -H 'Authorization: Bearer {access_token}' \
   -H 'Content-Type: application/json' \
   -d '{
-    "name": "Sub account name"
+    "legal_name": "legal business name"
   }'
 ```
 
-Use the sub accounts API to [create a sub account](https://docs.justifi.tech/api-spec#tag/Sub-Accounts/operation/CreateSubAccount) on JustiFi that is associated with your platform. You will need the `account_id` from the account you create for the next step.
+Use the entities API to [create a business](https://docs.justifi.tech/api-spec#tag/Business/operation/CreateBusiness) on JustiFi that is associated with your platform. You will need the `business_id` from the business you create for the next step.
 
 ### Include JustiFi hosted onboarding in your application
-To present the JustiFi hosted onboarding form to your user, create an iframe with a source of `https://accounts.justifi.ai/onboarding/SUB_ACCOUNT_ID`, where `SUB_ACCOUNT_ID` is the `account_id` that was created in the previous step. This iframe will present your user with a multi-step form where they can enter the business and financial information needed for approval. Upon submission, a success message will display.
+To present the JustiFi hosted onboarding form to your user, create an iframe with a source of `https://accounts.justifi.ai/onboarding/BUSINESS_ID`, where `BUSINESS_ID` is the `business_id` that was created in the previous step. This iframe will present your user with a multi-step form where they can enter the business and financial information needed for approval. Upon submission, a success message will display.
+
+(*Note: Passing a `sub_account_id` to the iframe instead of a `business_id` is still supported but will be deprecated soon*)
```

Comment:
I couldn't figure out how to link to our old hosted onboarding document without exposing it in the index (left side). I think this is a good compromise - acknowledging that there used to be a different way to use it but not confusing new customers).
Thoughts @pete2786 ?

Reply:
Works for me